src/test/java/com/google/sps/MockSVS.java

```diff
@@ -0,0 +1,66 @@
+package com.google.sps;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import java.util.HashMap;
+import java.util.Map;
+
+/* A shell class that has basic functionality of video search without integration tools */
+public class MockSVS {
```

Comment:
what is SVS? you might want to expand them.

Reply:
Supposed to be SingleVideoSearch to match class names, but I can change it for clarity